cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/outlet.json

```diff
@@ -15,5 +15,12 @@
     "horizontal-datum": "NAD84",
     "vertical-datum": "NAVD88"
   },
-  "rating-group-id": "Rating-location-outlet"
+  "rating-category-id": {
+    "office-id": "SPK",
+    "name": "Rating"
+  },
+  "rating-group-id": {
+    "office-id": "SPK",
+    "name": "Rating-location-outlet"
```

Comment:
Is this "Rating-" + <projectid> + "-outlet"? or a static string constant?

Reply:
It's an arbitrary string, and my unit tests go with "Rating-{location id}-outlet" for this particular test.

Reply:
@adamkorynta did you need any additional information on this PR?

Reply:
the test example should be a location name instead of just "location" then to avoid confusion.

Reply:
Updated the location ids made in here